flax/linen/attention.py

```diff
@@ -25,10 +25,10 @@
 import jax.numpy as jnp
 import numpy as np
 
-from .linear import default_kernel_init
-from .linear import DenseGeneral
-from .module import Module, compact
-from . import initializers
+from .linear import default_kernel_init  # pytype: disable=pyi-error
```

Comment:
Out of curiosity: what error does this give? It seems to trigger always for relative imports?

Reply:
This is some kind of weird pytype bug. I'll make a seperate tracking bug for this